### PR TITLE
docs: update uno.extension docs

### PR DESCRIPTION
**Pull request by nickrandolph**
Publishing extensions docs changes: https://github.com/unoplatform/uno.extensions/pull/2107

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by morning4coffe-dev — APPROVED** [review]

**Comment by jeromelaban** [comment]
@nickrandolph looks good to me, merge when ready.

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.0

File: doc/articles/common-issues-vs2022.md [hunk 1]
```diff
@@ -67,11 +67,6 @@ Event handlers [cannot be automatically](https://github.com/unoplatform/uno/issu
 
 A workaround is to use the [`x:Bind` to events feature](features/windows-ui-xaml-xbind.md#examples). This feature allows to use a simpler syntax like `<Button Click="{x:Bind MyClick}" />` and declare a simple method `private void MyClick() { }` in the code-behind.
 
-### Abnormally long build times with using WebAssembly and WSL
```
Comment (nickrandolph) [inline doc/articles/common-issues-vs2022.md:70]: @jeromelaban it seems as part of the re-org of the docs that "get-started-with-linux.md" has been removed with the content placed in other files. I couldn't see the content for downgrading to WSL 1 - is this still an appropriate fix? If so, need to add this section back with appropriate link

Comment (jeromelaban) [reply doc/articles/common-issues-vs2022.md:70]: There's no need to downgrade to WSL 1 anymore, and we're not suggesting to use WSL explicitly, only in very specific legacy cases.

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**